python/tvm/topi/arm_cpu/mprofile/dsp/micro_kernel/avg_pool.py

```diff
@@ -55,7 +55,7 @@ def _body():
             ib = tvm.tir.ir_builder.create()
             ib.emit(
                 tvm.tir.call_extern(
-                    cc.dtype,
+                    "int32",
```

Comment:
It is intentional, yes.  The `"int32"` is the return type of a PackedFunc from the PackedFunc interface.  Previously, the return type was the datatype of the output `cc` tensor, which isn't necessarily `"int32"`.  It didn't cause an error, because it was part of an `Evaluate` node and was never used.  However, with the additional forward declarations of functions within the IRModule, this was caught as a type error.